bot/dnd.go

```diff
@@ -0,0 +1,61 @@
+package bot
+
+import (
+	"fmt"
+	"log"
+	"time"
+	"venova/db"
+)
+
+func playCmd(ctx CommandCtx) error {
+	msg := ctx.Message
+	args := ctx.Args
+	// Set dnd play date
+	if msg.Author.ID == morthisId {
+		layout := "01-02-2006"
+		t, err := time.Parse(layout, args[1])
+		if err != nil {
+			return fmt.Errorf("error parsing date: %w", err)
+		}
+		currRoleId := getMemberDNDRole(msg.Member)
+		if currRoleId == "" {
+			log.Printf("Role not found.")
+			if err := ctx.Reply("Your dnd role is not found in the db."); err != nil {
+				return err
+			}
+		}
+		if err := db.InsertPlayDate(t, currRoleId); err != nil {
+			return fmt.Errorf("error inserting into table %w", err)
+		}
+		if err = ctx.Reply("The Date has been updated."); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func whenIsDndCmd(ctx CommandCtx) error {
+	msg := ctx.Message.Message
+	now := time.Now()
+	currRoleId := getMemberDNDRole(msg.Member)
+	if currRoleId == "" {
+		log.Printf("Could not find Dnd Role")
+	}
+	dateOfPlay, _, err := db.GetLatestPlayDate(currRoleId)
+	if err != nil {
+		if err := ctx.Reply("Could not find play date information. Perhaps wrong server."); err != nil {
+			return err
+		}
```

Comment:
Stop doing these useless if err checks pls & ty.

```suggestion
		return ctx.Reply("Could not find play date information. Perhaps wrong server.")
```